.storybook/themes/news.scss

```
@@ -1056,6 +1056,26 @@
 				single-chain-children-stack: (
 					gap: var(--global-spacing-6),
 				),
+				small-manual-promo: (
+					components: (
+						media-item: (
+							flex: 1 0 min-content,
+						),
+						image: (
+							display: inline,
+							aspect-ratio: attr(aspect-ratio),
```

This is not working as intended - you need to specify the entire attribute name. It is also not working due to the `Image` component not outputting the HTML attribute

```suggestion
							aspect-ratio: attr(data-aspect-ratio),
```

---

Confirmed. The `<Image>` component doesn't have that prop or a `...rest` so I can't send it in.
I believe we were using this for testing. I'll confirm that we need it.

---

It should work to maintain the aspect ration as long as one dimension is constrained... but the image currently doesn't pass along "...rest" like the other components (Was going to see if @matthewroach knew if there was a specific reason or it was missed, since he was the one that initially added them)

---

Looks like the image component wasn't updated with the rest - the only reason I would say it wasn't done was because it's a barebones component until we have resizer v2 logic added - I do not see a reason to not add `...rest` into the component for flexibility

---

@matthewroach , @nschubach : I've taken out usage of the `imageRatio` custom field for now pending availability of Image Resizer 2.0.